src/jit/hwintrinsicxarch.cpp

```diff
@@ -262,14 +275,45 @@ GenTree* Compiler::impSSE41Intrinsic(NamedIntrinsic intrinsic, CORINFO_METHOD_HA
 
 GenTree* Compiler::impSSE42Intrinsic(NamedIntrinsic intrinsic, CORINFO_METHOD_HANDLE method, CORINFO_SIG_INFO* sig)
 {
+    GenTree*  retNode  = nullptr;
+    GenTree*  op1      = nullptr;
+    GenTree*  op2      = nullptr;
+    var_types callType = JITtype2varType(sig->retType);
+
+    CORINFO_ARG_LIST_HANDLE argLst = sig->args;
+    CORINFO_CLASS_HANDLE    argClass;
+    CorInfoType             corType;
     switch (intrinsic)
     {
         case NI_SSE42_IsSupported:
-            return gtNewIconNode(compSupports(InstructionSet_SSE42));
+            retNode = gtNewIconNode(compSupports(InstructionSet_SSE42));
+            break;
+
+        case NI_SSE42_Crc32:
+            assert(sig->numArgs == 2);
+            op2 = impPopStack().val;
+            op1 = impPopStack().val;
+#if defined(_TARGET_X86_) && !defined(_TARGET_AMD64_)
```

Comment:
This should just be `#if defined(_TARGET_X86_)`

Reply:
Done.